### PR TITLE
docs(perf): refactor ComponentControls

### PR DESCRIPTION
**Pull request by layershifter**
### Before

image

### After

image

---

This PR grabs controls from Stardust. I asked my collegues about this, before this change they don't know about editor 🤔 

This also allows us to omit `Popup`s creation for each example, it costs over 50-100 ms in development mode.

### CODE REVIEW

File: docs/src/components/ComponentDoc/ComponentControls/ComponentControlsEditCode.js [hunk 1]
```diff
@@ -2,20 +2,16 @@ import PropTypes from 'prop-types'
...
+  </Menu.Item>
 )
 
 ComponentControlsEditCode.propTypes = {
   active: PropTypes.bool,
   onClick: PropTypes.func,
 }
 
-export default updateForKeys(['active'])(ComponentControlsEditCode)
+export default ComponentControlsEditCode
```
Comment (layershifter) [inline docs/src/components/ComponentDoc/ComponentControls/ComponentControlsEditCode.js:17]: Now, there is no sence in this HOCs.

**Comment by levithomason** [comment]
>This PR grabs controls from Stardust. I asked my collegues about this, before this change they don't know about editor 🤔

This is exactly what prompted the refactor here!  I was quite surprised and bummed out that people didn't know about the editor :(

**Comment by levithomason** [comment]
Should we also reverse the order of the links as in Stardust?  This makes the first link "Try it".

**Review by levithomason — APPROVED** [review]
See question, make a decision, and merge 👍

**Comment by levithomason** [comment]
On second thought, we can always reorder them after this is merged 😄

**Pull request merged**